.gitignore

```diff
@@ -2,6 +2,7 @@
 *.iml
 *.ipr
 *.iws
+nb*
```

Comment:
@marcosnils  I develop on top of netbeans
Didn't want to commit jedis configuration files